modules/otel-config-receive-metrics-monitoring-stack.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * observability/otel/otel-configuring-metrics-for-monitoring-stack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="configuration-for-receiving-metrics-from-the-monitoring-stack_{context}"]
+= Configuration for receiving metrics from the monitoring stack
+
+A configured OpenTelemetry Collector custom resource (CR) can configure the Prometheus receiver to scrape metrics from the in-cluster monitoring stack.
```

Comment:
Using "configured" and "configure" in the same sentence is a bit awkward. I think you could improve it by saying something like this:

```suggestion
By configuring an OpenTelemetry Collector custom resource (CR), you can enable the Prometheus receiver to scrape metrics from the in-cluster monitoring stack.
```

Reply:
Thank you for suggesting a rewording here. I replaced the second instance of "configure" with "set up".